### PR TITLE
Feat: Contact us section

### PR DESCRIPTION
**Pull request by Delyc**
## Describe your changes
This branch develops contact us section on desktop
## Issue ticket number and link
- ID: #015
- Link: https://www.notion.so/apeunit/Contact-Us-Section-78ab040eebb34187b27a7f6764ec7948

## Tasks completed
- [x] Contact us section on desktop
 
## Tasks not completed
- Contact us responsiveness on other devices

## Screenshots (if needed)

Desktop:

image

### CODE REVIEW

File: components/sections/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { LargeButton } from "../ui/Button";
+import { H3 } from "../ui/Headings";
+
+const ContactUs = () => {
+    return (
+        <section className="bg-primary-800 mx-0.5 rounded-8xl gap-16 xl:px-16 px-5 py-8 flex flex-col items-center">
+            <div className="xl:w-239">
+                <H3 text="Ready to talk about your project?" styles=" text-center text-white" />
```
Comment (alain-kubwayo) [inline components/sections/ContactUs.jsx:8]: You should remove the unnecessary white space when passing `styles` prop

File: components/sections/ContactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                <p className="text-white/90  text-center  ">Are you looking for a consultancy partnership to address your blockchain challenges? We{"’"}d love to hear from you.  Let{"’"}s build something <br />
```
Comment (alain-kubwayo) [inline components/sections/ContactUs.jsx:9]: You should also remove white spaces for readability and consistency purposes

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job, I left some comments.

**Review by Ntwali-Josue — APPROVED** [review]
Looks good 🚀

File: components/sections/ContactUs.jsx (hunk 2 above)
Comment (Delyc) [reply components/sections/ContactUs.jsx:9]: Done

File: components/sections/ContactUs.jsx (hunk 1 above)
Comment (Delyc) [reply components/sections/ContactUs.jsx:8]: Done

**Review by mhenriette — APPROVED** [review]
looks good!!

**Review by alain-kubwayo — APPROVED** [review]

**Pull request merged**